pythonFiles/include/blender_vscode/operators/addon_update.py

```diff
@@ -19,8 +19,15 @@ def execute(self, context):
             send_dict_as_json({"type" : "disableFailure"})
             return {'CANCELLED'}
 
-        for name in list(sys.modules.keys()):
-            if name.startswith(self.module_name):
```

Comment:
@JacquesLucke I looked at the PR and can not decipher what it is trying to do. I think we can close it as was here for years.
I agree that check is a bit too incluside
```
if name.startswith(self.module_name):
```
this one will avoid some unfortunate deletions:
```
if name == self.module_name or name.startswith(self.module_name + "."):
```
If you accept this I will make a PR.

Reply:
That sounds reasonable.